packages/nextjs/package.json

```diff
@@ -11,28 +11,40 @@
   },
   "main": "build/cjs/index.server.js",
   "module": "build/esm/index.server.js",
-  "browser": "build/esm/index.client.js",
   "types": "build/types/index.types.d.ts",
   "exports": {
     "./package.json": "./package.json",
     ".": {
+      "edge": {
+        "default": "./build/esm/edge/index.js"
+      },
+      "edge-light": {
+        "default": "./build/esm/edge/index.js"
+      },
+      "worker": {
+        "default": "./build/esm/edge/index.js"
+      },
+      "workerd": {
+        "default": "./build/esm/edge/index.js"
```

Comment:
These are always esm only?

Reply:
Fair point, in general I think nextjs can cope with only having esm but I'm gonna try to add cjs exports too.